Resources/Public/JavaScript/GitHubLinkHandler.js

```diff
@@ -10,36 +10,17 @@
  *
  * The TYPO3 project - inspiring people to share!
  */
+import LinkBrowser from "@typo3/backend/link-browser.js";
+import RegularEvent from "@typo3/core/event/regular-event.js";
 
-/**
- * Module: TYPO3/CMS/Examples/GitHubLinkHandler
- * Github issue link interaction
- */
-define(['jquery', 'TYPO3/CMS/Recordlist/LinkBrowser'], function($, LinkBrowser) {
-	'use strict';
-
-	/**
-	 *
-	 * @type {{}}
-	 * @exports T3docs/Examples/GitHubLinkHandler
-	 */
-	var GitHubLinkHandler = {};
-
-	$(function() {
-		$('#lgithubform').on('submit', function(event) {
-			event.preventDefault();
-
-			var value = $(this).find('[name="lgithub"]').val();
-			if (value === 'github:') {
-				return;
-			}
-			if (value.indexOf('github:') === 0) {
-				value = value.substr(7);
-			}
-			LinkBrowser.finalizeFunction('github:' + value);
-		});
-	});
-
-	return GitHubLinkHandler;
-});
+class GithubLinkHandler {
```

Comment:
As the file is named "GitHubLinkHandler.js" with a capital "H" in "Hub" the class name should reflect this.